ait/core/table.py

```diff
@@ -504,14 +503,14 @@ def getDefaultDict():  # noqa: N802
 
 def YAMLCtor_FSWColDefn(loader, node):  # noqa: N802
     fields = loader.construct_mapping(node, deep=True)
-    return createFSWColDefn(**fields)
+    return FSWColDefn(**fields)
 
 
 def YAMLCtor_FSWTabDefn(loader, node):  # noqa: N802
     fields = loader.construct_mapping(node, deep=True)
-    fields['fswheaderdefns'] = fields.pop('header', None)
-    fields['coldefns'] = fields.pop('columns', None)
-    return createFSWTabDefn(**fields)
+    fields["fswheaderdefns"] = fields.pop("header", None)
+    fields["coldefns"] = fields.pop("columns", None)
+    return FSWTabDefn(**fields)
```

Comment:
Sorry, my comment from last review was incorrect. The `createFSWxxx` method gets created in run-time during the extension process (i.e. `util.__init_extensions__`). So it is okay to call `createFSWxxx` method here. Please revert back to before this commit.

Reply:
Ok, I made the changes back and update utll.py